### DISCUSSION TITLE
which backends support read_parquet?

### DISCUSSION THREAD

**Discussion by wakamex** [ibis-project/ibis#8455] [Q&A] [answered]
can I find out without installing each backend and trying it? I was hoping I'd see this in the [operation support matrix](https://ibis-project.org/support_matrix), but it doesn't have any rows for "read" or "parquet" in them! I guess those don't count as "operations" 🤨

**Comment by lostmygithubaccount** [comment, marked as answer]
perhaps not the ideal UX, but right now you can check each backend's documentation to see if `read_parquet` is there, e.g. for ClickHouse: https://ibis-project.org/backends/clickhouse#ibis.backends.clickhouse.Backend.read_parquet

**Comment by csubhodeep** [comment]
xref #9448
